docs/source/app_development/index.rst

```diff
@@ -0,0 +1,135 @@
+=============================
+Develop AiiDAlab applications
+=============================
+
+############################################################
+Variant A: Quickstart using the AiiDAlab application cutter
+############################################################
+
+Open the terminal from the AiiDAlab start page by clicking on the icon labeled "Terminal":
+Then in the terminal, type:
+
+   .. code-block:: console
+
+       ~$ cd apps
+       ~/apps$ cookiecutter https://github.com/aiidalab/aiidalab-app-cutter.git
+       app_name [template]:
+       ...
+
+After answering the questions asked by the app-cutter, you will find a new folder inside the `apps` directory that contains the basic structure of an app.
+The newly created app will show up with a rendered banner on the start page after a reload.
+
+**Note** AiiDAlab expects that app source code is managed with `git <https://git-scm.com/>`.
+To start tracking the app development with git, open the terminal once more, and execute:
+
+   .. code-block:: console
+
+       ~$ cd apps/my-app/                              # enter the app directory
+       ~/apps/my-app$ git init                         # initialize git repository
+       ~/apps/my-app$ git add -A                       # add all files
+       ~/apps/my-app$ git commit -m "initial commit"   # create initial commit
+
+#################################################
+Variant B: Create app from scratch (step-by-step)
+#################################################
+
+Typically one would prefer to use the cookie-cutter to create a new app, but here are the steps to create an app from scratch.
+
+Step 1: Create the app folder
+=============================
+
+Open the terminal and then create a new app with:
+
+   .. code-block:: console
+
+       ~$ git init apps/my-app
+
+After this, just update the home page and you will see that your new app is already there!
+However, it is obvious that there are still a few essential things missing: specifically the app metadata including a title and description.
+
+.. _fig_app_development_new_app:
+.. figure:: include/new_app.png
```

Comment:
The screenshot must be updated, since we directly initialized the git repository, it should say "Modified", not "Unable to determine...".

Reply:
updated in 733f130